### PR TITLE
WebTab: Save/Restore zoom level on restart

### LINKED ISSUE #1768: please save/restore the current zoom level on restart
It would be nice if qupzilla could remember the current zoom level on restarts.

### PR DESCRIPTION
**Pull request by innermous**
Closes #1768 .

### CODE REVIEW

File: src/lib/webtab/webtab.cpp [hunk 1]
```diff
@@ -73,8 +74,8 @@ QDataStream &operator <<(QDataStream &stream, const WebTab::SavedTab &tab)
     stream << tab.url;
     stream << tab.icon.pixmap(16);
     stream << tab.history;
+    stream << tab.zoomLevel;
     stream << tab.isPinned;
-
```
Comment (nowrep) [inline src/lib/webtab/webtab.cpp]: Unrelated whitespace change

File: src/lib/webtab/webtab.cpp [hunk 2]
```diff
@@ -289,7 +300,6 @@ bool WebTab::isRestored() const
 void WebTab::restoreTab(const WebTab::SavedTab &tab)
 {
     Q_ASSERT(m_tabBar);
-
```
Comment (nowrep) [inline src/lib/webtab/webtab.cpp]: ditto

File: src/lib/webtab/webtab.cpp [hunk 3]
```diff
@@ -321,7 +332,6 @@ void WebTab::restoreTab(const WebTab::SavedTab &tab)
 void WebTab::p_restoreTab(const QUrl &url, const QByteArray &history)
 {
     m_webView->load(url);
-
```
Comment (nowrep) [inline src/lib/webtab/webtab.cpp]: ditto

File: src/lib/webtab/webtab.cpp [hunk 4]
```diff
@@ -91,7 +92,7 @@ QDataStream &operator >>(QDataStream &stream, WebTab::SavedTab &tab)
     stream >> tab.url;
     stream >> pixmap;
     stream >> tab.history;
-
```
Comment (nowrep) [inline src/lib/webtab/webtab.cpp]: ditto

File: src/lib/webtab/webtab.cpp (hunk 3 above)
Comment (innermous) [inline src/lib/webtab/webtab.cpp]: I see. Just forgot to clean... I'll update PR in a sec.
